src/js/models/v2/recipient/BaseRecipientOverview.js

```diff
@@ -16,6 +16,7 @@ const BaseRecipientOverview = {
         this._totalTransactions = parseFloat(data.total_transactions) || 0;
         this.businessTypes = data.business_types || [];
         this.level = data.recipient_level || 'R';
+        this.recipientHash = data.recipientHash || null;
```

Comment:
Also updated in [this PR](https://github.com/fedspendingtransparency/usaspending-website/pull/838). Once it's merged, you can merge dev into your branch to test.